hack/releaser/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.19-openshift-4.12
-RUN yum -y install hub && yum clean all
+RUN curl -OL https://github.com/github/hub/releases/download/v2.14.2/hub-linux-amd64-2.14.2.tgz && \
```

Comment:
Please add a TODO with a pointer to the problem so we can revert back to yum install once fixed.

Reply:
Comment added